bylaws.tex

```diff
@@ -307,7 +307,7 @@
 If the member is still unsatisfied after being heard by the Executive Board, the appeal may be brought to the attention of the ResLife Advisor.
 
 \bsection{Expectations of House Members}
-Each member is required to pay House dues as stated in \ref{Collection of House Dues}, attend all House Meetings, and attend at least twenty-five (25) of the permanent director meetings.
+Each member is required to pay House dues as stated in \ref{Collection of House Dues}, attend all House Meetings, and attend at least twenty-five (25) of the House director meetings (including Ad-Hoc director meetings).
```

Comment:
While we're at it, maybe `s/director/directorship/` to match above and be phrased a little better?